app/views/users/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<!-- <%= form_tag(user_path, method: :put)  do |form| %>
+    <%= label_tag "Usernamer" %>
+    <%= text_field_tag :username, @user.username %><br>
```

Comment:
- [ ] you should have another type of form commented here as well

Reply:
it can be done in the edit form because we can't use put method in a html form, so we put the other type form into the new.html.erb
```
<!-- <form action="/users" method="post">
    First name: <input type="text" name="username"><br>
    Email: <input type="text" name="email"><br>
    password: <input type="text" name="password"><br>
    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
    <input type="submit" value="Submit">
</form> -->